pkg/pmem-csi-driver/parameters/parameters.go

```diff
@@ -114,6 +116,7 @@ type Volume struct {
 	Persistency *Persistency
 	Size        *int64
 	VolumeID    *string
+	DeviceMode  *string
```

Comment:
Let's make this type-safe:
- merge the operator branch
- use `* api.DeviceMode` as type

Reply:
I was keeping this intentionally as a string, because we get this value from `DeviceManager.GetName()` which returns a string.

Reply:
That also should be changed.

Reply:
Done.